core/src/modules/reset.ts

```diff
@@ -28,6 +28,7 @@ export namespace Reset {
    * * clears the redis cache
    * * clears resque
    */
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I left these here because the test is definitely passing a callerId. But callerId is not used in any of these functions. I could update the usage of these so that callerId is no longer used and leave it for some future use.

Reply:
🤷 Probably not worth removing